### ISSUE TITLE
Angular: group by module

### ISSUE THREAD

**Issue by raDiesle** [manland/draw-my-project#77]
Like in this simple but great project, grouping by module is very powerful,
if you grouped your angular by module, so not by type.

See Code below:
https://github.com/carlo-colombo/grunt-angular-modules-graph/blob/master/tasks/angular_modules_graph.js

See some suggested improvements, which are almost already implemented for draw-my-project below:
https://github.com/carlo-colombo/grunt-angular-modules-graph/issues

**Comment by manland** [comment]
Hello!

Thanks for your interest in this project. We are currently on a new version (no open-source for now). This feature is in our backlog and will be implemented soon.

The new version of draw-my-project contains a lot of bug fixes, new features, advice... If you want to test it, just let me know and i will add you in the alpha-testers list.

**Comment by raDiesle** [comment]
Yes, please, I would be a good candidate, and if it is okay, I will use it indirect/direct for upcoming public talks

**Comment by raDiesle** [comment]
So the new version will be commercial, implemented under business circumstances, or what?
Is there a sense/way to get involved in this great project, too?

**Comment by manland** [comment]
Ok soon you will receive mail (of your github account let me know if you want to change) with steps to test draw-my-project v2. 

Before this, you can show all you want, but please don't link to this old repo (i will communicate to you the new url soon (on the end of next week). 

The new project will be in business circumstances if the feedback of alpha-testers and beta-testers are good.

We are on the end of the alpha release so you can't help us for now. But if you want  help us for beta test or production release, we will mail a message to start a community (but after the alpha-test probably in september).
